projects/ngneat/error-tailor/src/lib/providers.ts

```diff
@@ -24,6 +24,11 @@ export type ErrorTailorConfig = {
   blurPredicate?: (element: Element) => boolean;
   controlErrorComponent?: Type<ControlErrorComponent>;
   controlErrorComponentAnchorFn?: (hostElement: Element, errorElement: Element) => () => void;
+  controlErrorsOn?: {
```

Comment:
Please change to:

```ts
{
  controlErrorsOn: {
     async: boolean;
     blur: boolean;
     change: boolean;
  }
}

```

Reply:
updated. I kept them optional, is that ok?